pkg/scheduler/factory/factory.go

```diff
@@ -884,24 +691,7 @@ func (c *configFactory) deletePodFromSchedulingQueue(obj interface{}) {
 }
 
 func (c *configFactory) invalidateCachedPredicatesOnUpdatePod(newPod *v1.Pod, oldPod *v1.Pod) {
```

Comment:
Is there a reason this function isn't deleted?

Reply:
Emm, I will blame the latest rebase :-)